samples/VanillaJSTestApp/app/b2c/authConfig.js

```diff
@@ -0,0 +1,27 @@
+// Config object to be passed to Msal on creation
+const msalConfig = {
+    auth: {
+        clientId: "e3b9ad76-9763-4827-b088-80c7a7888f79",
+        authority: "https://login.microsoftonline.com/tfp/msidlabb2c.onmicrosoft.com/B2C_1_SISOPolicy/",
+        knownAuthorities: ["login.microsoftonline.com"]
+    },
+    cache: {
+        cacheLocation: "localStorage", // This configures where your cache will be stored
+        storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
+    },
+    system: {
+        telemetry: {
```

Comment:
Telemetry doesn't need to be included in these tests.

Reply:
Good call, missed that when I was copied the default sample